src/lib/stores/contactsStore.js

```diff
@@ -83,6 +83,15 @@ function createContactsStore() {
                 return false
             }
         },
+
+        /**
+         * Checks if the given address is a contract address
+         * @param {string} address Address to check
+         * @returns {boolean} True if the address is a contract address, false otherwise
+         */
+        isContractAddress: (address) => {
```

Comment:
This is a really helpful function to have included. Thanks!

Reply:
You are welcome :)